src/managers/input/InputManager.cpp

```diff
@@ -1066,23 +1066,40 @@ void CInputManager::setPointerConfigs() {
             libinput_device_config_accel_set_speed(LIBINPUTDEV, LIBINPUTSENS);
 
             const auto ACCELPROFILE = g_pConfigManager->getDeviceString(devname, "accel_profile", "input:accel_profile");
+            const auto SCROLLPOINTS = g_pConfigManager->getDeviceString(devname, "scroll_points", "input:touchpad:scroll_points");
```

Comment:
why is this in touchpad: if it's going to apply globally? you don't check for a touchpad.

Reply:
Idk why I thought that would make sense, it also works for mice. Will fix.